packages/opc-feedback/src/opc-feedback.ts

```diff
@@ -157,13 +157,13 @@ export class OpcFeedback extends LitElement {
           required></textarea>
           <p class="op-feedback__subtitle pf-u-font-size-xs">
           ${this.template.bugSubmissionNote}</p>
-        <button class="pf-c-button pf-m-block ${this._error.length === 0 ? 'pf-u-display-none-on-sm' : ''}"
+        <button class="pf-c-button pf-m-block ${(this._error.length && this._summary.length) === 0 ? 'pf-u-display-none-on-sm' : ''}"
```

Comment:
What type of condition do you want to check here? Because I'm not sure it would work as you'd expect here.
```js
0 && 0        // > 0
0 && 1        // > 0
1 && 0        // > 0
1 && 1        // > 1
```
So if you want to check if both of the lengths are 0, then you should probably use `||`. Because the condition (`&&`) it would be true if any one of those is 0.